packages/web/src/dev/ValidatorsListStyles.tsx

```diff
@@ -7,6 +7,11 @@ export const styles = StyleSheet.create({
     position: 'static',
     zIndex: 'initial',
   } as any,
+  pStaticOverflow: {
+    position: 'static',
+    zIndex: 'initial',
+    overflow: 'initial',
```

Comment:
curious which of these is causing it to need to be typed as any?

Reply:
The "initial" value seems not supported